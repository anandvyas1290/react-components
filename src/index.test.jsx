import React from "react";
import renderer from "react-test-renderer";
import { describe, expect, it } from "vitest";
import { Button } from "./index";

describe("Greeting component", () => {
    it("component renders correctly", () => {
        const component = renderer.create(<Button />);

        const tree = component.toJSON();

        expect(tree).toMatchSnapshot();
    });

    it("prop working", () => {
        const component = renderer.create(<Button label={"Universe"} />);

        const tree = component.toJSON();

        expect(tree).toMatchSnapshot();
    });
});
