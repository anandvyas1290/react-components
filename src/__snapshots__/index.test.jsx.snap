// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Greeting component > component renders correctly 1`] = `
<button
  className="px-4 border-fuchsia-500 py-2.5 flex justify-center rounded-lg2 font-semibold text-gray-50 bg-rwc-primary1 text-white hover:bg-opacity-95 transition-all duration-300 undefined"
/>
`;

exports[`Greeting component > prop working 1`] = `
<button
  className="px-4 border-fuchsia-500 py-2.5 flex justify-center rounded-lg2 font-semibold text-gray-50 bg-rwc-primary1 text-white hover:bg-opacity-95 transition-all duration-300 undefined"
>
  Universe
</button>
`;
